djangocms_page_sitemap/sitemap.py

```diff
@@ -31,14 +31,13 @@ def items(self):
             .filter(language__in=languages, path__isnull=False, page__login_required=False)
```

Comment:
We need to prefetch page__page_content, then in the url loop below (for page_url in all_urls:) we can exclude any urls that don't have a page_content entry.